src/patternfly/components/Select/select.scss

```diff
@@ -82,24 +82,41 @@
   --pf-c-select__menu-item--PaddingTop: var(--pf-global--spacer--sm);
   --pf-c-select__menu-item--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-select__menu-item--m-selected--PaddingRight: var(--pf-global--spacer--2xl);
+  --pf-c-select__menu-item--m-selected--m-link--PaddingRight: var(--pf-global--spacer--lg);
   --pf-c-select__menu-item--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-select__menu-item--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-select__menu-item--FontSize: var(--pf-global--FontSize--md);
   --pf-c-select__menu-item--FontWeight: var(--pf-global--FontWeight--normal);
   --pf-c-select__menu-item--LineHeight: var(--pf-global--LineHeight--md);
   --pf-c-select__menu-item--Color: var(--pf-global--Color--dark-100);
+  --pf-c-select__menu-item--Width: 100%;
   --pf-c-select__menu-item--disabled--Color: var(--pf-global--Color--dark-200);
   --pf-c-select__menu-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
   --pf-c-select__menu-item--focus--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
   --pf-c-select__menu-item--disabled--BackgroundColor: transparent;
+  --pf-c-select__menu-item--m-link--Width: auto;
+  --pf-c-select__menu-item--m-link--hover--BackgroundColor: transparent;
+  --pf-c-select__menu-item--m-link--focus--BackgroundColor: transparent;
+  --pf-c-select__menu-item--m-action--hover--BackgroundColor: transparent;
+  --pf-c-select__menu-item--m-action--Color: var(--pf-global--disabled-color--200);
+  --pf-c-select__menu-item--m-action--Width: auto;
+  --pf-c-select__menu-item--m-action--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-select__menu-item--hover__menu-item--m-action--Color: var(--pf-global--Color--200);
+  --pf-c-select__menu-item--m-action--hover--Color: var(--pf-global--Color--100);
+  --pf-c-select__menu-item--m-action--focus--Color: var(--pf-global--Color--100);
+  --pf-c-select__menu-wrapper--m-favorite__menu-item--m-favorite-action--Color: var(--pf-global--palette--gold-400);
 
   // Menu item icon
   --pf-c-select__menu-item-icon--Color: var(--pf-global--active-color--100);
   --pf-c-select__menu-item-icon--FontSize: var(--pf-global--icon--FontSize--sm);
   --pf-c-select__menu-item-icon--Right: var(--pf-global--spacer--md);
+  --pf-c-select__menu-item--m-link__menu-item-icon--Right: #{pf-size-prem(2px)};
   --pf-c-select__menu-item-icon--Top: 50%;
   --pf-c-select__menu-item-icon--TranslateY: -50%;
 
+  // Menu item action icon
+  --pf-c-select__menu-item-action-icon--MinHeight: calc(var(--pf-c-select__menu-item--FontSize) * var(--pf-c-select__menu-item--LineHeight));
```

Comment:
should the height be based off of `pf-c-select__menu-item-main` instead since that is now what the icon is adjacent to?